docs/hello_nf-core/01_run_demo.md

```diff
@@ -183,15 +183,23 @@ params {
 }
 ```
 
-This tells us that the `nf-core/demo` test profile already specifies the input parameter, so you don't have to provide any input yourself.
-However, the `outdir` parameter is not included in the test profile, so we will have to add it to the execution command using the `--outdir` flag.
+The test profile shows us what has been pre-configured for testing: most notably, the `input` parameter is already set to point to a test dataset, so we don't need to provide our own data.
+
+The comment block at the top also includes a usage example showing how to run with this test profile.
+Notice that it includes `--outdir <OUTDIR>` - this tells us we'll need to specify an output directory when we run the pipeline.
 
 ### 2.2. Run the pipeline
 
-Our examination of the test profile above told us what pipeline argument(s) we need to specify: just `--outdir`.
+Based on the usage example in the test profile, we know we need to specify `--outdir` to tell the pipeline where to save results.
 
 We're also going to specify `-profile docker,test`, which by nf-core convention enables the use of Docker containers, and of course, invokes the test profile.
 
+!!! note "Understanding container profiles"
+
+    The `-profile docker` option tells Nextflow to use Docker containers for running processes.
+    nf-core pipelines are designed to work with containers (Docker, Singularity, etc.) to ensure reproducibility and eliminate software installation issues.
+    The profile system allows you to easily switch between different container engines or execution environments.
```

Comment:
Should we add a link to where we explain this in the Hello Nextflow section 6 on Configuration?